Documentation/ProjectFile/prj/processes/process/THERMO_MECHANICAL_PHASE_FIELD/phasefield_parameters/t_history_field.md

```diff
@@ -0,0 +1 @@
+A damage-driving history field which is associated with the maximum local tensile strain energy. It is only used in monolithic scheme.
```

Comment:
Since the monolithic scheme is removed, then the history field parameter can be removed too?

Reply:
the parameter history_field is removed.